lib/nsxv.py

```diff
@@ -503,30 +505,83 @@ def _TranslatePolicy(self, pol, exp_info):
           term = self.FixHighPorts(term, af=af)
           if not term:
             continue
-          new_terms.append(Term(term, filter_type, 4))
+          new_terms.append(Term(term, filter_type, applied_to, 4))
 
         if filter_type == 'inet6':
           af = 'inet6'
           term = self.FixHighPorts(term, af=af)
           if not term:
             continue
-          new_terms.append(Term(term, filter_type, 6))
+          new_terms.append(Term(term, filter_type, applied_to, 6))
 
         if filter_type == 'mixed':
           if 'icmpv6' not in term.protocol:
             inet_term = self.FixHighPorts(term, 'inet')
             if not inet_term:
               continue
-            new_terms.append(Term(inet_term, filter_type, 4))
+            new_terms.append(Term(inet_term, filter_type, applied_to, 4))
           else:
             inet6_term = self.FixHighPorts(term, 'inet6')
             if not inet6_term:
               continue
-            new_terms.append(Term(inet6_term, filter_type, 6))
+            new_terms.append(Term(inet6_term, filter_type, applied_to, 6))
 
-      self.nsxv_policies.append((header, filter_name, [filter_type],
+      self.nsxv_policies.append((header, filter_name, [filter_type], section_id,
                                  new_terms))
 
+  def _ParseFilterOptions(self, filter_options):
+    """Parses the target in header for filter type, section_id and applied_to.
+       target:: nsxv mixed 12345 securitygroup securitygroupId
```

Comment:
Please see other doc strings or the following link on how to construct a docstring
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

You need to create the sections for arguments, returns and errors and correctly format them